### DISCUSSION TITLE
Contributions not appearing for work

### DISCUSSION THREAD

**Discussion by Lexx426** [community/community#119998] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi everyone! I would first like to apologize if this is not an actual issue. 

I volunteer with a nonprofit and so far have committed to their repository a total of 10 times. But only 1 time, has it popped up as a contribution on my chart and on my contribution activity. They both have separate branches, now deleted since that work is done, and were merged into the newest release that we are working on. Is this a bug or why would all my commits not appear at all? 

I am new to contributing to someone else's repo so I am sorry if there is something that I am not understanding. Thank you all for your help and patience. 

![image](https://github.com/community/community/assets/95107297/3030eedf-4772-4b60-840a-fd8b2984a561)

**Comment by fdistasio** [comment]
GitHub only counts contributions made to the main branch. If your commits were made to different branches and were merged into the default branch, they should still count as contributions. However, if they were made to branches other than the default one, they might not show up on your profile.

**Reply by Lexx426** [reply]
So, once that other branch/sprint is merged into the default branch, my contributions should show then?

**Reply by fdistasio** [reply]
Yes, that's correct

**Reply by fdistasio** [reply]
If you received the response you wanted, please close this discussion marking an answer.

**Comment by Kyushu21** [comment]
Visibility Settings.
Ensure that the repository's visibility settings allow contributions to be counted towards your profile. Sometimes, repositories are set to not count contributions from outside collaborators.

**Reply by Lexx426** [reply]
Okay, thank you, I will connect with the owner of the repository to double check that for me!
